src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/data.html.twig

```diff
@@ -21,6 +21,10 @@
  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- *#}
+ * #}
```

Comment:
```suggestion
 *#}
```

Reply:
Also, don't add spaces here, or I believe our `prestashop:licenses:update` command will not detect the license correctly and it will add another one